From: Marcel Holtmann <marcel@holtmann.org>
Date: Tue, 11 Mar 2014 09:33:31 -0700
Subject: hwdb: Update database of Bluetooth company identifiers

(cherry picked from commit f401e2b24ba17452f9655993a6dedefdffe06d23)
(cherry picked from commit 873fbb5c7a36c2553aee6db2c9afbbbba94ca879)
---
 hwdb/20-bluetooth-vendor-product.hwdb | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/hwdb/20-bluetooth-vendor-product.hwdb b/hwdb/20-bluetooth-vendor-product.hwdb
index 0335a71..7d86f10 100644
--- a/hwdb/20-bluetooth-vendor-product.hwdb
+++ b/hwdb/20-bluetooth-vendor-product.hwdb
@@ -911,3 +911,15 @@ bluetooth:v012C*
 
 bluetooth:v012D*
  ID_VENDOR_FROM_DATABASE=Sony Corporation
+
+bluetooth:v012E*
+ ID_VENDOR_FROM_DATABASE=ASSA ABLOY
+
+bluetooth:v012F*
+ ID_VENDOR_FROM_DATABASE=Clarion Co., Ltd.
+
+bluetooth:v0130*
+ ID_VENDOR_FROM_DATABASE=Warehouse Innovations
+
+bluetooth:v0131*
+ ID_VENDOR_FROM_DATABASE=Cypress Semiconductor Corporation
